client/spa-pages/components/MapPage.tsx

```diff
@@ -153,12 +153,6 @@ const MapInner = ({ setPage }: Props) => {
 	};
 	const forceUpdate = useForceUpdate();
 
-	///// Keeping for similar implementations /////
-	// const handleStickyFooter = () => {
-	// 	setIsExpanded(!isExpanded);
-	// 	setFacCardIsOpen(false);
-	// };
-

```

Comment:
I always prefer deleting the code and removing it using VCS if we need it later. 
The main reasons are that it keeps the code looking a bit cleaner. So it's nothing major.